package.json

```diff
@@ -44,7 +44,7 @@
     "expect": "^1.13.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.8.2",
-    "flexboxgrid": "^6.3.0",
+    "flexboxgrid": "Zetoff/flexboxgrid#hidden-columns",
```

Comment:
Hi @silvenon yeah: this is temporary because it depends on this flexboxgrid branch https://github.com/kristoferjoseph/flexboxgrid/pull/211